docusaurus/docs/metadata.md

```diff
@@ -0,0 +1,252 @@
+---
+id: reference-plugin-json
+title: Reference (plugin.json)
+description: "Reference for the plugin.json metadata file."
```

Comment:
nit: I don't think we need the " marks https://docusaurus.io/docs/markdown-features/head-metadata

Reply:
fixed